packages/wallet-ui/src/components/ui/molecule/AmountInput/AmountInput.view.tsx

```diff
@@ -110,29 +103,29 @@ export const AmountInputView = ({
   };
 
   const handleMaxClick = () => {
-    if (inputRef.current && asset.usdPrice) {
+    if (value && asset.usdPrice) {
```

Comment:
not quite sure if this condition is correct, it look like u only do it when value is exist or > 0?